Program.cs

```diff
@@ -1,28 +1,39 @@
-using System;
-using System.Collections.Generic;
+using BoldBIEmbedSample.Data;

```

Comment:
Remove the unwanted files from project like csproj.user file, bin, obj and package-lock.json.

Reply:
Unwanted files are removed.